components/consumers/pdf/Dockerfile

```diff
@@ -1,11 +1,19 @@
-FROM golang:latest
 
-WORKDIR /playwright
-RUN go mod init github.com/ocurity/pdf-consumer &&\
-    go get -u github.com/playwright-community/playwright-go &&\ 
-    go run github.com/playwright-community/playwright-go/cmd/playwright@latest install --with-deps
+# Stage 2: Build
+FROM golang:latest as builder
```

Comment:
Nit: any reason why the images are not pinned?

In my experience it's nicer to pin them so we can gate potential upgrades :)